src/SidePanel.tsx

```diff
@@ -21,22 +21,32 @@ import { getConfigArray } from './features/Configuration/boardControllerConfigSl
 export default () => {
     logger.debug('Rendering SidePanel');
 
+    const [isWriting, setWriting] = useState<boolean>(false);
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏Pretty sure typescript will infer the type here so no need to add it explicitly.